Gemfile

```diff
@@ -12,9 +12,8 @@ gem 'colorize'
 gem 'devise', '~> 4.7.3'
 gem 'dotenv-rails', '~> 2.7', '>= 2.7.6'
 gem 'enum_help'
-gem 'figaro'
 gem 'httparty'
-gem 'identity-hostdata', git: 'https://github.com/18F/identity-hostdata.git', tag: 'v1.0.1'
+gem 'identity-hostdata', git: 'https://github.com/18F/identity-hostdata.git', tag: 'v3.1.0'
```

Comment:
I'll need to publish a new tag with [this change](https://github.com/18F/identity-hostdata/pull/27) and add it here before merge